pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -4997,7 +4997,7 @@ __ri void GSRendererHW::EmulateTextureSampler(const GSTextureCache::Target* rt,
 	m_conf.ps.tcc = m_cached_ctx.TEX0.TCC;
 
 	m_conf.ps.ltf = bilinear && shader_emulated_sampler;
-	m_conf.ps.point_sampler = g_gs_device->Features().broken_point_sampler && !target_region && (!bilinear || shader_emulated_sampler);
+	//m_conf.ps.point_sampler = g_gs_device->Features().broken_point_sampler && GSConfig.GPUPaletteConversion && !target_region && (!bilinear || shader_emulated_sampler);
```

Comment:
Is it normal that this line is commented? That would completely disable the workaround on DX/OGL, while according to the PR message it should still be enabled with the additional `GSConfig.GPUPaletteConversion` check.

Reply:
If you read the commit name you would see it's a test.

Reply:
Ah indeed sorry